test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -299,3 +299,24 @@ Describe "Connection" -Tag "CI", "RequireAdminOnWindows" {
         Test-Connection $UnreachableAddress -TcpPort 80 -TimeOut 1 | Should -BeFalse
     }
 }
+
+Describe "Test-Connection should run in the default synchronization context (threadpool)" -Tag "CI" {
+    It "Test-Connection works after constructing a WindowsForm object" {
```

Comment:
Skip if not Windows

Reply:
Right, will update. Not sure why `Update-Help` tests are failing though 😕

Reply:
>  Not sure why Update-Help tests are failing though 

It seems we use TCPClient in here and there. Maybe we don't cleanup?

Reply:
Uh, I see we have the same in #11518 and in scheduled daily tests.

Reply:
Yeah, not sure what change causes the help tests to fail ...

Reply:
@SteveL-MSFT your comment was addressed, please take another look when you get a chance.

Reply:
Help content got changed and the fwlinks updated. They should pass now.